ReleaseNotes.md

```diff
@@ -1,224 +1,348 @@
-* [FLUID-5717: Merge policies contributed via dynamic grades do not operate correctly](https://issues.fluidproject.org/browse/FLUID-5717)
-* [FLUID-5615: Grade closure algorithm for dynamic grades is faulty](https://issues.fluidproject.org/browse/FLUID-5615) (partial fix)
-* [FLUID-5742: Dynamic components do not respond to dynamic `typeName` or `gradeNames`](https://issues.fluidproject.org/browse/FLUID-5742)
-* [FLUID-5694: Diagnostic when injecting component to its own location](https://issues.fluidproject.org/browse/FLUID-5694)
-* [FLUID-5696: Failure to gingerly instantiate injection records](https://issues.fluidproject.org/browse/FLUID-5696)
-* [FLUID-5664: Renderer fails to respect rebinding of model reference](https://issues.fluidproject.org/browse/FLUID-5664)
-* [FLUID-5671: Memory exhaustion possible during `fluid.fail` on node.js (`fluid.prettyPrintJSON`)](https://issues.fluidproject.org/browse/FLUID-5671)
-* [FLUID-5582: Improve flexibility of `fluid.fail` handling further](https://issues.fluidproject.org/browse/FLUID-5582)
-* [FLUID-5667: Circularity in material sent for options expansion is not detected](https://issues.fluidproject.org/browse/FLUID-5667)
-* [FLUID-5711: The Infusion core framework doesn't work when run in a Web Worker](https://issues.fluidproject.org/browse/FLUID-5711)
-* [FLUID-5509: Storing NaN via a "new-style applier" will cause infinite recursion in DataBinding](https://issues.fluidproject.org/browse/FLUID-5509)
-* [FLUID-5675: Text To Speech enactor should be supplied with a mock implementation to facilitate integration testing](https://issues.fluidproject.org/browse/FLUID-5675)
-
-### Removals
+For a complete list of Fixes and Improvements see the [Version 2.0](https://issues.fluidproject.org/projects/FLUID/versions/10041) summary in the [JIRA](https://issues.fluidproject.org) issue tracker.
 
-* [FLUID-5616: "init functions" removed from the framework](https://issues.fluidproject.org/browse/FLUID-5616)
-* Also removed: the static and dynamic environments, dynamic invokers, old ChangeApplier, autoInit, etc.
-
-## What's New in 1.9 ##
-
-**Note** 1.9 was never (so far) an official release of Infusion, but was a "last ditch" branching point to record the state of the framework should
-we ever in future require to make a release or backport fixes to a version of Infusion API compatible with the latest 1.x line, most particularly 1.5.x.
-It is stored in github at [Infusion 1.9.x](https://github.com/fluid-project/infusion/tree/1.9.x)
-
-This list of JIRAs is nonexhaustive since work has accumulated in trunk for several years since the 1.5 Infusion release. These are some of the more significant recent fixes,
-currently not including work on the preferences framework:
+**Note:** Infusion 1.9 was never (so far) an official release of Infusion, but was a "last ditch" branching point to record the state of the framework should
```

Comment:
I don't really know what this "'last ditch' branching point" statement means, and I don't think many users will be able to interpret it. Can we say this is in simpler terms?

"Infusion 1.9 was not officially released, but is available as an official branch so that we can backport fixes to a version of the Infusion API compatible with the latest 1.x line."

Or something along these lines?